endpoints/api/billing.py

```diff
@@ -40,6 +40,19 @@
 import uuid
 import json
 
+MILLISECONDS_IN_SECONDS = 1000
+
+
+def check_internal_api_for_subscription(namespace_user):
+    """
+    If Reconciler is enabled, RH marketplace should also be checked
+    for valid subscriptions
```

Comment:
Is this comment still valid, or is this independent of the reconciler?

Reply:
Should be independent, changed it to be more accurate